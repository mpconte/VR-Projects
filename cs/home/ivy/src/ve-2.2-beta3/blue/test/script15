#
# Escape handling test
#
[stdout] writeln Hello\nWorld
[stdout] writeln "Hello\nWorld"
