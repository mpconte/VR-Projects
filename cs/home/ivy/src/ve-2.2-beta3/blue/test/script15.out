Hello\nWorld
Hello
World
result = ok
